wages_challenge.sh

```diff
@@ -0,0 +1,28 @@
+#Gets unique combinations of gender and yearsExperience, sorting by gender and then yearsExperience
+echo "Unique combos of gender and yearsExp, sorted:"
+cat wages.csv | grep -v "gender" | cut -d ',' -f 1,2 | sed 's/,/ /g' | sort -k1,1 -k2n | uniq
+
+#---Start of challenge 2---
```

Comment:
Correct, but returns the years schooling column as well. Please read directions fully.